public/images/backgrounds/top_desktop.svg

```diff
@@ -1,14 +1,33 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="840" height="63" viewBox="0 0 840 63">
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="840px" height="63px" viewBox="0 0 840 63" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 53.2 (72643) - https://sketchapp.com -->
+    <title>Elements/Background-search/Desktop/Maps</title>
+    <desc>Created with Sketch.</desc>
     <defs>
-        <path id="a" d="M746.279 49.105A123.753 123.753 0 0 1 760.045 0H838c-25.457 18.424-56.627 35.001-91.721 49.105z"/>
+        <filter x="-2.7%" y="-36.5%" width="105.5%" height="173.0%" filterUnits="objectBoundingBox" id="filter-1">
+            <feOffset dx="0" dy="2" in="SourceAlpha" result="shadowOffsetOuter1"></feOffset>
+            <feGaussianBlur stdDeviation="5" in="shadowOffsetOuter1" result="shadowBlurOuter1"></feGaussianBlur>
+            <feColorMatrix values="0 0 0 0 0   0 0 0 0 0   0 0 0 0 0  0 0 0 0.15 0" type="matrix" in="shadowBlurOuter1" result="shadowMatrixOuter1"></feColorMatrix>
+            <feMerge>
+                <feMergeNode in="shadowMatrixOuter1"></feMergeNode>
+                <feMergeNode in="SourceGraphic"></feMergeNode>
+            </feMerge>
+        </filter>
+        <path d="M746.279657,49.1054378 C747.453039,31.4813556 752.2939,14.8607401 760.04583,6.57252031e-14 L838.001118,1.50576946e-05 C812.543666,18.4242705 781.374434,35.0011886 746.279655,49.1054609 Z" id="path-2"></path>
     </defs>
-    <g fill="none" fill-rule="nonzero">
-        <path fill="#76BC54" d="M197.718 63c-.475-4.435-.718-8.94-.718-13.5 0-17.578 3.614-34.312 10.14-49.5h230.72C444.387 15.188 448 31.922 448 49.5c0 4.56-.243 9.065-.718 13.5H197.718z"/>
-        <path fill="#4BA2EA" d="M288.603 0a216.031 216.031 0 0 1-21.69 63H71C46.494 56.715 23.028 49.44-.001 42V0h288.604z"/>
-        <path fill="#65CEE7" d="M183.714 0a177.173 177.173 0 0 1-27.819 63H71C46.494 56.715 23.028 49.44-.001 42V0h183.715z"/>
-        <path fill="#87C966" d="M519 0h319c-34.382 24.883-79.182 46.397-130 63-83.84-.014-146.84-.014-189 0V0z"/>
-        <path fill="#87C966" d="M386.542 63c-12.283-18.494-20.505-39.918-23.393-63h292.702a146.867 146.867 0 0 1-3.364 17.853A146.746 146.746 0 0 1 632.458 63H386.542z"/>
-        <path fill="#9DDC7D" d="M472.013 63c-13.805-17.773-22.881-39.403-25.34-63h246.654a123.566 123.566 0 0 1-3.121 17.56c-4.25 16.771-11.919 32.179-22.22 45.44H472.014z"/>
-        <use fill="#76BC54" xlink:href="#a"/>
+    <g id="Elements/Background-search/Desktop/Maps" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Group" filter="url(#filter-1)" transform="translate(0.998882, 0.000000)">
```

Comment:
this extra "translate" causes a blank border on the left, and should be removed.

Reply:
WOW !!!